docs/dev/fundamentals/reading-games-code.md

```diff
@@ -0,0 +1,97 @@
+---
+description: This article teaches how to use ILSpy in order to read game's code
+---
+
+# Reading game's code
```

Comment:
As mentioned before - please use Title Case - and consider renaming the article an file to "Reading Game-Code" or something similar